dev-python/flask-wtf/flask-wtf-0.14.3.ebuild

```diff
@@ -0,0 +1,52 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8} )
+inherit distutils-r1
+
+MY_PN="Flask-WTF"
+MY_P="${MY_PN}-${PV}"
+
+DESCRIPTION="Simple integration of Flask and WTForms"
+HOMEPAGE="https://pythonhosted.org/Flask-WTF/ https://pypi.org/project/Flask-WTF/"
+SRC_URI="mirror://pypi/${MY_P:0:1}/${MY_PN}/${MY_P}.tar.gz"
+S="${WORKDIR}/${MY_P}"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="amd64 x86"
+
+distutils_enable_sphinx docs
+
+IUSE="doc test"
+
+RESTRICT="!test? ( test )"
+
+RDEPEND="
+	dev-python/Babel[${PYTHON_USEDEP}]
+	dev-python/flask[${PYTHON_USEDEP}]
+	dev-python/flask-babel[${PYTHON_USEDEP}]
+	dev-python/itsdangerous[${PYTHON_USEDEP}]
+	dev-python/jinja[${PYTHON_USEDEP}]
+	dev-python/werkzeug[${PYTHON_USEDEP}]
+	>=dev-python/wtforms-1.0.5[${PYTHON_USEDEP}]
+"
+DEPEND="
```

Comment:
BDEPEND. Also why not d_e_t?

Reply:
Also done, but tests are failing. They are also failing for the current stable package. I'd like to get this PR merged (since it fixes pgadmin4 in ~arch) and then spend time on fixing tests.